docs/conf.py

```diff
@@ -283,7 +296,7 @@
 epub_title = u'Nextflow'
 epub_author = u'Paolo Di Tommaso'
 epub_publisher = u'Paolo Di Tommaso'
-epub_copyright = u'2013-2018, Centre for Genomic Regulation'
+epub_copyright = u'2013-2024, Seqera Labs, Centre for Genomic Regulation'
```

Comment:
I think the copyright can just be Seqera, @pditommaso to confirm

Reply:
True - this is what's used elsewhere. I'll just copy this:

https://github.com/nextflow-io/nextflow/blob/744fff2a7753b75b0125caf073ad4b30a89433ef/build.gradle#L2